### DISCUSSION TITLE
PR on forks

### DISCUSSION THREAD

**Discussion by scarf005** [leonsteinhaeuser/project-beta-automations#47] [Q&A] [answered]
since [actions triggered from forks doesn't send secrets](https://github.com/orgs/community/discussions/26163), `pull_request_target` instead of `pull_request` needs to be used. However then i'm unable to fetch `${{ github.event.pull_request.node_id }}`.
what should i do to use this action with PR on forks?

- [full workflow log](https://github.com/exciting-transcendence/transcendence/runs/7718258983?check_suite_focus=true)
- [related discuttion](https://github.com/orgs/community/discussions/24709)
- [difference in github contexts on various triggers](https://github.com/cloud-barista/cb-coffeehouse/wiki/The-github-context-output-by-cases-for-GitHub-Actions-workflows)

**Comment by leonsteinhaeuser** [comment, marked as answer]
I found a resource on the internet that seems to solve this problem. I haven't tested this yet, but if you have time and can share your results, we can add this to the documentation.

See: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

**Reply by scarf005** [reply]
oh, it was that i forgot to change to `pull_request_target` in bash script below.
now it works on forked repositories as well!
[workflow file](https://github.com/exciting-transcendence/transcendence/blob/3647ca80b708ab00bf7161dde5a514ffa0ec76a1/.github/workflows/project_board.yml)
[workflow log](https://github.com/exciting-transcendence/transcendence/actions/runs/2817526553)

**Reply by leonsteinhaeuser** [reply]
Great, thanks for your feedback.

**Reply by scarf005** [reply]
i think i could create a PR to add this to README, should i file a issue?

**Reply by leonsteinhaeuser** [reply]
Yeah, please do so. Thanks in advance
